pages/introduction.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <section class="h-full px-9 pb-8 overflow-hidden" :class="{'bg-spotify-green': step === 0}">
+    <transition
+        mode="out-in"
+        :name="transition">
+      <div v-if="step === 0" class="pt-24 flex flex-col h-full justify-between">
+        <h1 class="text-4xl leading-[2.75rem] max-w-[12ch] font-bold">Hi, <br/>Welcome to what am I listening to.</h1>
+        <button @click="nextStep" class="bg-spotify-black py-4 rounded-full font-bold text-base inline-flex items-center justify-center gap-[0.875rem]">
+          <span>Continue</span>
+          <ArrowRight class="w-4 aspect-square"/>
+        </button>
+      </div>
+      <div v-else-if="step === 1" class="mx-2 relative text-center grid place-content-center h-full">
+        <div>
+          <img src="../assets/images/step-1.png" class="rounded-lg max-h-[288px] mx-auto"/>
+          <h1 class="mt-9 mb-4 font-bold text-2xl">I'm always trying to find out more about myself</h1>
+          <p>what better window into your <br/>emotions is there than your listening habits?</p>
+        </div>
+        <button @click="nextStep" class="bg-spotify-green absolute bottom-0 w-full py-4 rounded-full font-bold text-base inline-flex items-center justify-center gap-[0.875rem]">
+          <span>Continue</span>
+          <ArrowRight class="w-4 aspect-square"/>
+        </button>
+      </div>
+      <div v-else-if="step === 2" class="mx-2 relative text-center grid place-content-center h-full">
+        <div>
+          <img src="../assets/images/step-2.png" class="rounded-lg max-h-[288px] mx-auto"/>
+          <h1 class="mt-9 mb-4 font-bold text-2xl">Why don't you give a try? Analyse yourself.</h1>
+        </div>
+        <div class="absolute bottom-0 w-full flex items-center gap-4">
+          <button class="bg-gray p-3 rounded-full" @click="previousStep">
+            <ArrowRight class="rotate-180 transform w-6 aspect-square"/>
```

Comment:
Why did you choose vite-svg-importer instead of <svg src="...">?

Reply:
Instead of what? 🤔 I chose it basically because back than in Nuxt 3 Beta (before RC) the only available module to load SVGs as components was for vite. I have stayed with it until today.

Reply:
Oh, sorry for the incomplete question. For the /login page I just used a standard HTML `<img src="foobar.svg" alt="foobar"/>` tag. Adding the Vite SVG bundler also broke the SVGs on the /login page (obviously easily fixable, no big deal). Does the vite-svg-importer have benefits over the standard img tag?

Reply:
Sure, you can use SVGs how they should be used. This means u can modify the SVGs easily via css classes which would not be possible within an <img> tag. For example, when u look at the SVG I added in assets there's `fill="currentColor` which means I can assign any color I want via CSS.